src/base/projects/Browser.svelte

```diff
@@ -60,26 +69,45 @@
           );
 
     state = { status: Status.Loading, path };
-    try {
-      state = { status: Status.Loaded, path, blob: await promise, theme };
-      return state.blob;
-    } catch (err) {
-      console.warn("Could not load blob.");
-    }
+    state = { status: Status.Loaded, path, blob: await promise, theme };
+    return state.blob;
   };
 
+  onMount(() => {
+    browserErrorStore.set(undefined);
+  });
+
   // Get an image blob based on a relative path.
-  const getImage = async (imagePath: string): Promise<proj.Blob> => {
+  const getImage = async (
+    imagePath: string,
+  ): Promise<proj.Blob | undefined> => {
     const finalPath = utils.canonicalize(imagePath, path);
-    return project.getBlob(commit, finalPath, { highlight: false });
+    try {
+      return project.getBlob(commit, finalPath, { highlight: false });
+    } catch (e: any) {
```

Comment:
I think it's more correct to do `catch (e: unknown)`. But maybe we should remove it, since we don't use the error as far as I can see?

Reply:
Yeah removing it, doesn't work with prettier svelte plugin. But we can remove the type so I think it defaults to `unknown`

Reply:
d246e3c4

Reply:
This should work:
```diff
-    } catch (e: any) {
+    } catch {
```

Reply:
TIL 😅

Reply:
9f866d72